### DISCUSSION TITLE
Log rotation using fluenbit

### DISCUSSION THREAD

**Discussion by maitry-nutanix** [fluent/fluent-bit#7572] [Q&A]
Hi Team!!
Want to know whether fluentbit rotate logs or not for the log file created by file output plugin ?  I want to set a threshold limit of lets say 100MB for each log file created by file output plugin.  
If not then what are the ways to do it without loosing out on any logs ? 

Config used: 
```
input-tail-kube-system.conf: |
    [INPUT]
      Name tail
      Tag kube*
      Path /var/log/pods/*/*/*.log
      Parser docker
      refresh_interval 5
      mem_buf_limit 10MB
      DB /var/log/flb_kube.db
  output-file-kube-system.conf: |
    [OUTPUT]
      Name file
      Match *
      Mkdir true
      Path /var/log/kubelet
```

**Comment by patrick-stephens** [comment]
No rotation is carried out, you should manage this via whatever option is best for your environment e.g. `logrotate` on the host.

Possibly relevant: https://github.com/fluent/fluent-bit/pull/7541
